src/coreclr/nativeaot/Runtime/StackFrameIterator.cpp

```diff
@@ -1599,6 +1599,19 @@ void StackFrameIterator::UnwindUniversalTransitionThunk()
 #define STACK_ALIGN_SIZE 4
 #endif
 
+#if defined(TARGET_ARM64) && defined(__GNUC__)
+static inline void* PacStripPtr(void* ptr)
+{
+    __asm__ volatile (".arch_extension pauth\n"
+                      "xpaci %0"
+                      : "+r" (ptr)
```

Comment:
can we move this in `asmhelpers.S` and `asmhelpers.asm`?

Reply:
Done.